docs/dev-setup.md

```diff
@@ -222,6 +222,11 @@ $ yarn run test-e2e
 ```
 1. You can view the progress and results on [BrowserStack](https://www.browserstack.com/automate).
 
+### Simulator
+ANET has a simulator that can exercise of the functions. It is located in 'client/test/sim'. It works by interfacing with ANET through GraphQL queries. The simulator executes `stories` which are assigned to different user types and have different probabilities.   
+
+The simulator can be started by running 'yarn sim' in 'client'.
```

Comment:
Please use `yarn run sim`. Even though `yarn <script>` works, it is merely a convenient shorthand for `yarn run <script>` *provided* `<script>` isn't a built-in: https://yarnpkg.com/lang/en/docs/cli/run/#toc-yarn-run-script
So let's be explicit.